test/fw/ptl/utils/pbs_dshutils.py

```diff
@@ -216,6 +226,75 @@ def get_platform(self, hostname=None, pyexec=None):
         self._h2p[hostname] = splatform
         return splatform
 
+    def get_uname(self, hostname=None, pyexec=None):
```

Comment:
Hey @monkeystate. It might be the same case as the ptl_server.py file that pbs_dshutils.py file is also not synced with the master branch for you. Can you confirm whether this is also an intentional change or not?

Reply:
Unintentional change removed